.rubocop.yml

```diff
@@ -1488,3 +1489,9 @@ Style/WhileUntilDo:
 Style/YodaCondition:
   Enabled: true
   EnforcedStyle: forbid_for_all_comparison_operators
+
+Style/DisableSyntax:
+  DisableSyntax:
+    - unless
+    - ternary
```

Comment:
I would vote to keep ternary

Reply:
If we decide to eject ternaries (I'm on the fence), then I would have the fixes be in a fast-follow PR.